include/openmc/random_ray/random_ray.h

```diff
@@ -46,10 +46,20 @@ class RandomRay : public Particle {
   // Static data members
   static double distance_inactive_;          // Inactive (dead zone) ray length
   static double distance_active_;            // Active ray length
+  static double avg_miss_rate;               // Average ray miss rate per
+                                             // iteration for reporting
   static unique_ptr<Source> ray_source_;     // Starting source for ray sampling
   static RandomRaySourceShape source_shape_; // Flag for linear source
   static bool mesh_subdivision_enabled_;     // Flag for mesh subdivision
   static RandomRaySampleMethod sample_method_; // Flag for sampling method
+  static int64_t n_source_regions;             // Total number of source regions
+  static int64_t
+    n_external_source_regions; // Total number of source regions with
+                               // non-zero external source terms
+  static uint64_t total_geometric_intersections; // Tracks the total number of
+                                                 // geometric intersections by
+                                                 // all rays for reporting
+  static int negroups_;                          // Number of energy groups
```

Comment:
- The class variables should have a `_` at the end to follow the C++ class variable naming convention.

Reply:
There's also a "total_geometric_intersections" variable already in the `RandomRaySimulation` class. Perhaps we should just use that one instead (and make it static), rather than making a duplicate?